### PR TITLE
 concepts: Start documentation page for HA

### PR DESCRIPTION
**Pull request by glittershark**
Add the start of a documentation page for the current state of
HA (warm-read HA), including sections on how to run readyset in HA mode
and what HA guarantees

Refs: readysettech/readyset#516

### CODE REVIEW

File: pages/concepts/ha.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# High Availability
+
+For production deployments where the availability needs are greater than can be
+met by a single instance, ReadySet can be run in a configuration providing
+**High Availability**, or **HA**.
+
+## Running ReadySet in HA mode
+
+In order to run ReadySet in HA mode, you'll need a minimum of three ReadySet
+instances - one two run the server, and two (or more) to run the ReadySet
```
Comment (aglover) [inline pages/concepts/ha.mdx]: "one to run" rather than "one two run"

**Review by aglover — COMMENTED** [review]
One slight grammatical issue! This is super exciting!! Way to go!!

File: pages/concepts/ha.mdx (hunk 1 above)
Comment (glittershark) [reply pages/concepts/ha.mdx]: done

**Review by aglover — APPROVED** [review]
LGTM!

File: pages/concepts/_meta.json [hunk 2]
```diff
@@ -2,5 +2,6 @@
   "overview": "Key Concepts",
   "streaming-dataflow": "Streaming Dataflow",
   "efficiency": "Memory Effeciency",
+  "ha": "High Availability",
```
Comment (lukoktonos) [inline pages/concepts/_meta.json:5]: For some reason this isn't showing up for me on the preview by default, but it does if I manually navigate to the ha page: https://readyset-public-docs-archived-git-ha-docs-readyset.vercel.app/concepts/ha

image

image

File: pages/concepts/ha.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+For production deployments where the availability needs are greater than can be
+met by a single instance, ReadySet can be run in a configuration providing
+**High Availability**, or **HA**.
+
+## Running ReadySet in HA mode
+
+In order to run ReadySet in HA mode, you'll need a minimum of three ReadySet
+instances - one to run the server, and two (or more) to run the ReadySet
+adapters.
```
Comment (lukoktonos) [inline pages/concepts/ha.mdx]: nit: I think a sentence here like "The adapters will run in embedded-readers mode, meaning that the full result sets will be stored on the adapter instances, replicated across the number of adapters running" would help give context to the flag changes below.

File: pages/concepts/ha.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+[embedded-readers]: /reference/cli/readyset#--embedded-readers
+[haproxy]: https://www.haproxy.org/
+[elb]: https://aws.amazon.com/elasticloadbalancing/
+
+## What HA guarantees
+
+When running ReadySet in High Availability mode, any running, healthy adapter
+instance can accept connections over the PostgreSQL/MySQL wire protocols, and
+will serve cached queries from ReadySet cache nodes running *inside* the
+adapter. This means that cached *warm* reads in ReadySet will alwaays be
```
Comment (lukoktonos) [inline pages/concepts/ha.mdx]:
```suggestion
adapter. This means that cached *warm* reads in ReadySet will always be
```

typo

File: pages/concepts/_meta.json (hunk 2 above)
Comment (glittershark) [reply pages/concepts/_meta.json:5]: shows up locally for me: 
image
@daviddkkim is there anything else I need to do to get this to show up?

File: pages/concepts/ha.mdx (hunk 3 above)
Comment (glittershark) [reply pages/concepts/ha.mdx]: Done.

File: pages/concepts/_meta.json (hunk 2 above)
Comment (daviddkkim) [reply pages/concepts/_meta.json:5]: Usually when you redeploy on vercel, it reflects the new nav bar changes. There's some weird caching thing happening i think. Let me kick one off for you!

File: pages/concepts/ha.mdx (hunk 4 above)
Comment (glittershark) [reply pages/concepts/ha.mdx]: Done

File: pages/concepts/_meta.json (hunk 2 above)
Comment (daviddkkim) [reply pages/concepts/_meta.json:5]: @glittershark @lukoktonos it seems to be reflected correctly after the redeploy - let me know what yall think!

Comment (lukoktonos) [reply pages/concepts/_meta.json:5]: Yep can see it now in the preview--thanks!

**Review by lukoktonos — APPROVED** [review]

**Pull request merged**